redisinsight/api/src/modules/database/repositories/stack.databases.repository.ts

```diff
@@ -25,25 +26,26 @@ export class StackDatabasesRepository extends LocalDatabaseRepository implements
    * @inheritDoc
    */
   async exists(): Promise<boolean> {
-    return super.exists(REDIS_STACK_CONFIG.id);
+    return super.exists({} as SessionMetadata, REDIS_STACK_CONFIG.id);
   }
 
   /**
    * @inheritDoc
    */
   async get(
+    _: SessionMetadata,
     id: string,
     ignoreEncryptionErrors: boolean = false,
     omitFields: string[] = [],
   ): Promise<Database> {
-    return super.get(REDIS_STACK_CONFIG.id, ignoreEncryptionErrors, omitFields);
+    return super.get({} as SessionMetadata, REDIS_STACK_CONFIG.id, ignoreEncryptionErrors, omitFields);
```

Comment:
the same. Since we already have SessionMetadata let's pass it. There is no need to hardcode it in several places. Parent class should take care of this